.gitlab/packaging/oci.yml

```diff
@@ -23,7 +23,8 @@
     - ${CI_PROJECT_DIR}/tools/ci/retry.sh go install github.com/DataDog/datadog-packages/cmd/datadog-package@latest
     - OUTPUT_DIR="/tmp/oci_output"
     - mkdir -p ${OUTPUT_DIR}
-    - ls $OMNIBUS_PACKAGE_DIR
+    - export PACKAGE_VERSION=${PACKAGE_VERSION/.git/-git}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ is this sh magic that edits the ENV VAR value? Maybe it's because I'm not familiar with the syntax, but it could be nice to explicitly `sed` it so it's obvious for future readers (but if it's just me who didn't know you could do that, that's fine too).

Reply:
You guessed correctly, and fair enough, it's shorter but arguably less readable this way, I have no objection switching to `sed`

Reply:
done!